README.md

```diff
@@ -12,8 +21,10 @@ size).
 iotrace is based on [Open CAS Telemetry Framework (OCTF)](https://github.com/Open-CAS/open-cas-telemetry-framework). Collected traces are stored in OCTF trace
 location. Traces can later be converted to JSON or CSV format.
 
-iotrace consists of a kernel tracing module (iotrace.ko) and an executable
-(iotrace) with command line interface.
+The iotrace executable (iotrace command line application) includes an eBPF
+program which during tracing is loaded to the Linux kernel. The eBPF program
```

Comment:
during tracing is loaded to the Linux kernel -> is loaded to the Linux kernel during tracing